openslides_backend/models/meeting.py

```diff
@@ -16,10 +18,254 @@ class Meeting(Model):
+    )
+    motions_line_length = fields.IntegerField(minimum=40)
+    motions_reason_required = fields.BooleanField()
+    motions_enable_text_on_projector = fields.BooleanField()
+    motions_enable_reason_on_projector = fields.BooleanField()
+    motions_enable_sidebox_on_projector = fields.BooleanField()
+    motions_enable_recommendation_on_projector = fields.BooleanField()
+    motions_show_referring_motions = fields.BooleanField()
+    motions_show_sequential_number = fields.BooleanField()
+    motions_recommendations_by = fields.TextField()
+    motions_statute_recommendations_by = fields.TextField()
+    motions_recommendation_text_mode = fields.TextField(
+        enum=["original", "changed", "diff", "agreed"]
+    )
+    motions_default_sorting = fields.TextField(enum=["weight", "identifier"])
+    motions_identifier_type = fields.TextField(
+        enum=["per_category", "serially_numbered", "manually"]
+    )
+    motions_identifier_min_digits = fields.PositiveIntegerField()
+    motions_identifier_with_blank = fields.BooleanField()
+    motions_statutes_enabled = fields.BooleanField()
+    motions_amendments_enabled = fields.BooleanField()
+    motions_amendments_in_main_list = fields.BooleanField()
+    motions_amendments_of_amendments = fields.BooleanField()
+    motions_amendments_prefix = fields.TextField()
+    motions_amendments_text_mode = fields.TextField(
+        enum=["freestyle", "fulltext", "paragraph"]
+    )
+    motions_amendments_multiple_paragraphs = fields.BooleanField()
+    motions_supporters_min_amount = fields.IntegerField(minimum=0)
+    motions_supporters_enable_autoremove = fields.BooleanField()
+
+    motion_poll_default_type = fields.TextField()  # TODO: PollType enum
+    motion_poll_default_100_percent_base = fields.TextField()  # TODO: PercentBase enum
+    motion_poll_default_majority_method = (
+        fields.TextField()
+    )  # TODO: MajorityMethod enum
+    motion_poll_default_group_ids = fields.ManyToOneArrayField(
+        to=Collection("group"), related_name="used_as_motion_poll_default_id"
+    )
+    motion_poll_ballot_paper_selection = fields.TextField(
+        enum=["NUMBER_OF_DELEGATES", "NUMBER_OF_ALL_PARTICIPANTS", "CUSTOM_NUMBER"]
+    )
+    motion_poll_ballot_paper_number = fields.PositiveIntegerField()
+
+    motions_export_title = fields.TextField()
+    motions_export_preamble = fields.TextField()
+    motions_export_submitter_recommendation = fields.BooleanField()
+    motions_export_follow_recommendation = fields.BooleanField()
+
+    assignment_poll_default_type = fields.TextField()  # TODO: PollType enum
+    assignment_poll_default_method = fields.TextField()  # TODO: PollMethod enum
+    assignment_poll_default_100_percent_base = (
+        fields.TextField()
+    )  # TODO: PercentBase enum
+    assignment_poll_default_majority_method = (
+        fields.TextField()
+    )  # TODO: MajorityMethod enum
+    assignment_poll_default_group_ids = fields.ManyToOneArrayField(
```

Comment:
Please remove this setting here and in the list above. We should define this as boolean field in group model.